fmriprep/interfaces/multiecho.py

```diff
@@ -78,320 +74,44 @@ def _run_interface(self, runtime):
         return runtime
 
 
-class MaskT2SMapInputSpec(BaseInterfaceInputSpec):
-    image = File(mandatory=True, exists=True, desc='T2* volume to mask')
-    mask = File(mandatory=True, exists=True,
-                desc='skull-stripped mean optimal combination volume')
-    compress = traits.Bool(True, usedefault=True,
-                           desc='use gzip compression on .nii output')
-
-
-class MaskT2SMapOutputSpec(TraitedSpec):
-    masked_t2s = File(exists=True, desc='masked T2* map')
-
-
-class MaskT2SMap(SimpleInterface):
-    """
-    Masks an existing T2* map using the skull-stripped optimally combined
-    volume (i.e., a weighted combination of multi-echo data).
-
-    Example
-    =======
-    >>> from fmriprep.interfaces import multiecho
-    >>> mask_t2s = multiecho.MaskT2SMap()
-    >>> mask_t2s.inputs.image = 'sub-01_run-01_t2smap.nii.gz'
-    >>> mask_t2s.inputs.mask = 'sub-01_run-01_optcomb.nii.gz'
-    >>> res = mask_t2s.run() # doctest: +SKIP
-    """
-
-    input_spec = MaskT2SMapInputSpec
-    output_spec = MaskT2SMapOutputSpec
-
-    def _run_interface(self, runtime):
-        ext = '.nii.gz' if self.inputs.compress else '.nii'
-        flat_mask = apply_mask(self.inputs.image, self.inputs.mask)
-        masked_image = unmask(flat_mask, self.inputs.mask)
-        self._results['masked_t2s'] = fname_presuffix(
-            self.inputs.image, suffix='_masked' + ext, newpath=runtime.cwd, use_ext=False)
-        masked_image.to_filename(self._results['masked_t2s'])
-        return runtime
-
-
 class T2SMapInputSpec(BaseInterfaceInputSpec):
-    in_files = InputMultiPath(File(exists=True), mandatory=True,
+    in_files = InputMultiPath(File(exists=True), mandatory=True, minlen=2,
                               desc='multi-echo BOLD EPIs')
```

Comment:
If there's a minimum length of 2, then this should just be a `traits.List`, not an `InputMultiPath`.

Reply:
It looks like `InputMultiPath` this use in the [`FirstEcho` interface](https://github.com/poldracklab/fmriprep/blob/f5b502c329daef0a1d0766c1553d08cb65115af9/fmriprep/interfaces/multiecho.py#L34) as well -- should I update both ?

Reply:
Yes.